admin/pageaudit/report/templates/page_template.html

```diff
@@ -39,120 +37,70 @@
+                <a class="{{ templateHelpers.classes.navItem }} {% block menuHomeClass %}{% endblock %}" href="{% url 'plr:home' %}">Find reports</a>
+                <a class="{{ templateHelpers.classes.navItem }} {% block menuBrowsereportsClass %}{% endblock %}" href="{% url 'plr:reports_browse' %}">Browse reports</a>
+                <a class="{{ templateHelpers.classes.navItem }} {% block menuDashboardClass %}{% endblock %}" href="{% url 'plr:reports_dashboard' %}">Dashboard</a>
+                
+                {% get_flatpages as flatpages %}
+                {% for page in flatpages %}
+                    {% highlight_nav_item page.url as highlight_flag %}
+                    <a class="{{ templateHelpers.classes.navItem }} {% if highlight_flag %}pl-highlight{% endif %}" href="{{ FORCE_SCRIPT_NAME }}/report/pages{{ page.url }}">{{ page.title }}</a>
+                {% endfor %}
+            </nav>
+            
+            {% block leadspace %}
+                <div class="{{ templateHelpers.classes.grid }} pl-leadspace bg-black pv4">
+			        <h1 class="f2 mt0 mb3 {{ templateHelpers.classes.siteColor }}">{% block pageTitle %}{% endblock %}</h1>
+			        {% block pageSubtitle %}{% endblock %}
+			    </div>
+            {% endblock %}
             
-            {% if IBM_BRANDING %}
-                <!-- Footer -->
-                <footer role="contentinfo" aria-label="IBM"><div id="ibm-footer" class="ibm-fullwidth"><ul><li><a href="http://www.ibm.com/contact/us/en/">Contact</a></li><li><a href="http://www.ibm.com/privacy/us/en/">Privacy</a></li><li><a href="http://www.ibm.com/legal/us/en/">Terms of use</a></li><li><a href="http://www.ibm.com/accessibility/us/en/">Accessibility</a></li></ul></div></footer>
+            
+            {% if request.session.showMessage %}
+                <p class="{{ templateHelpers.classes.grid }} mb3 {{ request.session.showMessage.class }}">{{ request.session.showMessage.text|safe }}</p>
             {% endif %}
-        </div>
-
+            
+            
+            {% block content %} {% endblock %}
+            
+            
+            <p class="tr-ns mb3 mt5 tl {{ templateHelpers.classes.grid }}">Made with <br>
+                <a class="dib" style="transform:translate3d(0,-3px,0);" href="https://www.postgresql.org/" target="_blank"><img src="{% static 'report/img/postgresql-logo.gif' %}" width="120" height="25" title="PostgreSQL: The world's most advanced open source relational database" alt="Made with PostgreSQL"></a> <span class="dib" style="transform:translate3d(0,-12px,0)">+</span> <a class="dib" href="https://www.djangoproject.com/" target="_blank"><img src="{% static 'report/img/django-logo.gif' %}" width="100" height="35" title="Django makes it easier to build better Web apps more quickly and with less code." alt="Made with Django"></a></p>
```

Comment:
I do think we should add a node.js & redis badge here: maybe make them all smaller so they fit nicely?